src/DDTrace/OpenTelemetry/Span.php

```diff
@@ -69,7 +78,8 @@ private function __construct(
         ResourceInfo $resource,
         array $links = [],
         int $totalRecordedLinks = 0,
-        bool $isRemapped = true
+        bool $isRemapped = true,
+        array $events = [],
```

Comment:
```suggestion
        array $events = [],
        bool $isRemapped = true
```
The trailing comma is not allowed on old PHP versions, also, ordering.